plugins/modules/ec2_vpc_route_table.py

```diff
@@ -57,7 +57,7 @@
     description:
       - List of routes in the route table.
       - Routes are specified as dicts containing the keys V(dest) and one of V(gateway_id),
-        V(instance_id), V(network_interface_id), or V(vpc_peering_connection_id).
+        V(instance_id), V(network_interface_id), V(transit_gateway_id), or V(vpc_peering_connection_id).
```

Comment:
This looks like we might have actually supported this before (if you knew the undocumented key to add here).

Is it worth modifying the arg_spec definition to only accept "good" values?

Reply:
I thought the same. In case we do that we need to ensure backward compatibility or introduce a breaking change

Reply:
+1 I thought the same

Reply:
I think adding choices with specified values would be best. By doing so, we could add this PR to version 9.0.0, as it might be a breaking change. Am I right? cc @alinabuzachis